app/models/info_request_batch.rb

```diff
@@ -82,4 +99,22 @@ def self.send_batches
                                                        info_request_batch.user).deliver
     end
   end
+
+  # Build an InfoRequest object which is an example of this batch.
+  def example_request(embargo_duration=nil)
```

Comment:
Hmm, there totally were, I think I rebased them away by accident. Re-instated in 2c1de39